### PR TITLE
update pdcurses to 3.9

### PR DESCRIPTION
**Pull request by charlesseizilles**
...and fix terminal resizing on windows

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -9,7 +9,7 @@
 	url = ../../xmake-io/xmake-core-sv.git
 [submodule "core/src/pdcurses/pdcurses"]
 	path = core/src/pdcurses/pdcurses
-	url = ../../xmake-io/xmake-core-pdcurses.git
+	url = https://github.com/wmcbrine/PDCurses.git
```
[the remark this answers is not in the document]
Comment (charlesseizilles) [reply .gitmodules:12]: can you synchronize xmake-core-pdcurses's master branch with the official repo, please?

[the remark this answers is not in the document]
Comment (waruqi) [reply .gitmodules:12]: I haven't had time to sync some of the previous changes and I'm not sure they're still needed. https://github.com/xmake-io/xmake-core-pdcurses/commit/5a522dc7ac23b8571f68851b844d3dd4dc6a67a5

[the remark this answers is not in the document]
Comment (charlesseizilles) [reply .gitmodules:12]: I don't think those changes are needed anymore. Now I just need to merge the 3.9 version's commit on xmake-core, right?

[the remark this answers is not in the document]
Comment (waruqi) [reply .gitmodules:12]: you can try xmake-core2 branch first. if ci works, I will switch to xmake-core branch.

[the remark this answers is not in the document]
Comment (charlesseizilles) [reply .gitmodules:12]: nevermind, `set_warnings("none")` did the trick

**Pull request merged**